### PR TITLE
Framework: Remove usages of lodash split()

### PR DESCRIPTION
**Pull request by tyxla**
#### Changes proposed in this Pull Request

We have only several usages of lodash's `split()` method. `split()` is pretty general and can be replaced with its alternative `String.split()`. This PR addresses that, in order to make us a bit less dependent on lodash.

This PR should not offer any visual or functional changes.

#### Testing instructions

* Verify the changes make sense.
* Verify tests still pass.

### CODE REVIEW

**Review by sarayourfriend — COMMENTED** [review]
These _seem_ safe but I'd like at least one other person to take a look just to confirm that we're not missing anything.

**Comment by tyxla** [comment]
Fair enough, thanks for taking a look @saramarcondes 👍  Will wait for another review.

**Review by sarayourfriend — APPROVED** [review]
@tyxla I gave this another close review and confirmed that all the changes seem safe. I don't want to block this any further so as far as I'm concerned feel free to merge when you're ready!

**Comment by tyxla** [comment]
Thanks again @sarayourfriend! Much appreciated 🙌

**Pull request merged**
